netcontrold/lib/dataif.py

```diff
@@ -184,7 +184,7 @@ def add_rxq(self, _id):
         # check if this rxq is already available.
         rxq = self.find_rxq_by_id(_id)
         if rxq:
-            return rxq
+            raise ObjConsistencyExc("rxq %d not found" % _id)
```

Comment:
This change should go in a separate commit as it is fixing a problem in app. Create a new commit with the clear problem statement and what you fixed and resubmit. The reason we keep logically separated commit is to blame a change and revert only that commit in future if a need arise.
In this case, if I revert this commit, it will revert the test case as well. You got it ?. Thanks.

Reply:
ok